frontend/src/components/pages/RegistrantExperience/RegistrationSteps/index.tsx

```diff
@@ -76,8 +77,17 @@ const RegistrationSteps = ({
       optionalClauses: [],
     },
   ]);
+  const [
+    requireEarlyDropOffLatePickup,
+    setRequireEarlyDropOffLatePickup,
+  ] = useState<boolean | null>(null);
```

Comment:
wasn't sure how we wanted to handle the early drop off/ late pickup step so I just created a flag - this is very much a temp thing can change in this PR if preferred!

Reply:
I think the boolean or null is fine. It makes sense logically since the state should be null if EDLP is not offered